src/suspectplacesearcher/main.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+from geopy.distance import vincenty
+import requests
+import math
+
+
+class SuspectPlaceSearch(object):
+
+    ''' Return a dictonary containt information of a
+        suspect place arround a given position.
+        A suspect place is some company that match a suspect keyword
+        in a Google Places search.
+        input:
+            - lat : latitude
+            - lng : longitude
+        output: closest_suspect_place:
+            closest_suspect_place is a dict with the keys:
+                - name : The name of suspect place
+                - latitude : The latitude of suspect place
+                - longitude : The longitude of suspect place
+                - distance : The distance in meters between suspect place and the given lat, lng 
+                - address : The address  of suspect place
+                - phone : The phone of suspect place
+                - id : The Google Place ID of suspect place
+                - keyword : The Keyword that generate the place
+                            in Google Place Search
+    '''
+
+    def __init__(self, key):
+        self.GOOGLE_API_KEY = key
+
+    def search(self, lat, lng):
+
+        suspect_keywords = ["Acompanhantes", "Motel",
```

Comment:
Once again. I begin to try to construct a crawler to specialized sites on this kind of place (motel, prostitutes, sex massage, etc). But the problem is these websites no contains so much information for all Brazilians regions. But when I start I notice that these places are in google maps with soft descriptions. For example, "Bahamas" (A famous São Paulo Brothel)  are in Google Maps with the description "Adult Entertainment Club"  and this keyword is used for another's brothels in brazil.